sw/airborne/modules/ctrl/ctrl_eff_sched_rot_wing.c

```diff
@@ -57,143 +70,163 @@
 #error "NO ROT_WING_EFF_SCHED_M defined"
 #endif
 
-#ifndef ROT_WING_EFF_SCHED_ROLL_ARM
-#error "NO ROT_WING_EFF_SCHED_ROLL_ARM defined"
+#ifndef ROT_WING_EFF_SCHED_DM_DPPRZ_HOVER_PITCH
+#error "NO ROT_WING_EFF_SCHED_DM_DPPRZ_HOVER_PITCH defined"
```

Comment:
All these defines needs to be set in the `test` node of the module file to avoid errors.